public/js/nfts.js

```diff
@@ -56,11 +57,34 @@ function renderNFTs(query = '') {
   domTotalHolders.innerText = arrOwners.length.toLocaleString('en-GB');
 
   // Total Burns
-  domTotalBurns.innerText = nTotalBurns.toLocaleString('en-GB');
+  if (nTotalBurns > 0){
+  	domTotalBurns.innerText = nTotalBurns.toLocaleString('en-GB');
+  }
+  else{
+	getInfoCollection();
+  }
 }
 
 // Fetch and render the collection!
 $.getJSON('https://stakecubecoin.net/web3/scp/tokens/getcollection/' + id, function(res) {
     data = res;
     renderNFTs();
-});
\ No newline at end of file
+});
+
+function getInfoCollection(){
+	$.getJSON('https://stakecube.io/api/v2/marketplace/collection?contract=' + id, function(res) {
+		if (res.success){
+			var nftsSold = res.result.nftsSold.length;
+			var nftsForSale = res.result.nftsForSale.length;
+			
+			if (nftsSold > 0){
+				domTotalBurnsTxt.innerText = 'Total sold in the Marketplace';
```

Comment:
Why are the variables not re-named?

Reply:
If you are referring to domTotalBurnsTxt, the name is always the same because the id of the container div always remains the same